test/UnitTest/TestingRepositories/InstanceTests.cs

```diff
@@ -72,7 +74,48 @@ public async Task Instance_Update_Task_Ok()
             int count = await PostgresUtil.RunCountQuery(sql);
             Assert.Equal(1, count);
             Assert.Equal("Task_2", updatedInstance.Process.CurrentTask.ElementId);
-            Assert.NotEqual("Shouldn't be updated", updatedInstance.Process.CurrentTask.Name);
+            Assert.Equal(newInstance.Process.CurrentTask.Name, updatedInstance.Process.CurrentTask.Name);
+            Assert.Equal("After update", updatedInstance.Process.CurrentTask.Name);
```

Comment:
Could you add some asserts for properties we expect to remain unchanged?

Insert a value for `newInstance.Process.CurrentTask.AltinnTaskType` and check that it's the same in the updated instance.

Reply:
The PR reflects the functionality in the current code. When updating the Process object all properties are set from user input and/or controller logic. No info from the existing instance will survice.

Reply:
The bug we're trying to fix with this change was an issue with IsHardDeleted under Status disapearing after an update to IsArchived or vise versa. I want a test for this case.

Reply:
This is already covered on line 114